### DISCUSSION TITLE
Auto Import fix

### DISCUSSION THREAD

**Discussion by QaahirStewart** [nuxt/nuxt#18943] [Questions]
Since 2 updates back, my auto imports are being called in the script section. Need a fix for this, components are not working when the import is being called, works when i delete the import in script.

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by QaahirStewart** [reply]
I created a fresh build and found out when hitting tab to close/complete a tag or use a component . I'm finding  it imports it directly but when I just type it in it doesn't. I'm going to look at the extensions to see if the change is coming from there.
